orbyter-ml-dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# orbyter-ml-dev:3.3
+# orbyter-ml-dev:3.5
```

Comment:
what happened to `3.4`

Reply:
This comment never got updated. 3.4 does already exist, I just hadn't noticed the update.